packages/monaco/data/monaco-nls.json

```diff
@@ -0,0 +1,1379 @@
+{
```

Comment:
@msujew I assume the file was copied from vscode? Do you know if a past CQ covered the file, else I believe we should register one:
- https://github.com/eclipse-theia/theia/wiki/Registering-CQs#codecontent-is-copied-from-other-projectsrepos

Reply:
No, this file is generated by the `translate-monaco` script that I wrote. It combines the index based approach from monaco with the key-value pair based approach from the language packs, in order to localize monaco. This file is basically just a generated bridge to connect them both.

Reply:
Thank you for clarifying!